packages/sdk-react-native/src/env.ts

```diff
@@ -39,10 +39,15 @@ export const EnvVars = {
         return {};
     },
     RNV_SKIP_LINKING: () => {
+        const { platform } = getContext();
         const skipPlugins: string[] = [];
         parsePlugins(
             (plugin, pluginPlat, key) => {
-                if (pluginPlat.disabled || plugin.disabled) {
+                if (
+                    pluginPlat.disabled ||
+                    plugin.disabled ||
+                    (plugin.supportedPlatforms && !plugin.supportedPlatforms.includes(platform!))
```

Comment:
@ElenaDiachenko we want to avoid non-null assertion operators in this case `platform!` as it skips TS null checking guards

```
(plugin.supportedPlatforms && platform && !plugin.supportedPlatforms.includes(platform))
```